### PR TITLE
[feat-5457]: Add packages and initialize i18next with default json

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-5457](https://gemeente-amsterdam.atlassian.net/browse/SIG-5457)

### Context
This PR introduces i18next with a default json which serves as fallback when the backend doesn't return translations

note:
- the labels under `algemeen` are translated from the language setting of the app.base.json file because we want Dutch namespaces here. 

### Changes
- Add i18next and related packages
- Initialize i18next. 
- Fetch json from backend
- Add default json as fallback

### Testing
- backend test file serves a `test` key
- Add `test` key to the fall back json ("key":  "this is local fallback key)
- Implement this key somewhere using `  const { t } = useTranslation()`
- See the backend value appear
- when disabling this call, the fall back should appear

[SIG-5457]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5457?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/shared/services/i18n/fallback/locales/nl/translations.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "algemeen": {
+        "toestemming-delen-contactgegevens": "Ja, ik geef Signalen toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
+        "wijk": "Wijk",
+        "kop-titel": "",
```
Comment (alimpens) [inline src/shared/services/i18n/fallback/locales/nl/translations.json:5]: Do we need this?

File: src/shared/services/i18n/fallback/locales/nl/translations.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        "contact-body": "Mogen we u bellen voor vragen? En op de hoogte houden via e-mail?",
+        "waarom-contact": "Vaak hebben we nog een vraag. Daarmee kunnen we het probleem sneller of beter oplossen. Of we willen iets uitleggen. Wij willen u dan graag even bellen. Of anders e-mailen wij u.",
+        "privacy": "Wij gebruiken uw telefoonnummer en e-mailadres alléén voor deze melding.",
+        "doorsturen": "Mogen we uw melding doorsturen?",
+        "waarom-doorsturen": "Soms kan de gemeente niets doen. Een andere organisatie moet dan aan het werk. Als dat zo is kunnen wij uw melding soms doorsturen. Wij sturen uw telefoonnummer of e-mailadres mee. Maar dat doen we alleen als u dat goed vindt.",
+        "toestemming": "Ja, ik geef de gemeente Amsterdam toestemming om mijn melding door te sturen naar andere organisaties als de melding niet voor de gemeente is bestemd.",
+        "controleer": "Controleer uw gegevens en verstuur uw melding.",
+        "geen-gegevens": "U hebt geen contactgevens ingevuld. We kunnen u niet laten weten wat wij hebben gedaan met uw melding.",
+        "bedankt": "Bedankt",
+        "meldnummer": "Uw melding is bij bekend onder {{SIG}}.",
```
Comment (alimpens) [inline src/shared/services/i18n/fallback/locales/nl/translations.json:33]:
```suggestion
        "meldnummer": "Uw melding is bij ons bekend onder {{SIG}}.",
```

File: src/shared/services/i18n/fallback/locales/nl/translations.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
+{
+    "algemeen": {
+        "toestemming-delen-contactgegevens": "Ja, ik geef Signalen toestemming om mijn contactgegevens te delen met andere organisaties, als dat nodig is om mijn melding goed op te lossen.",
+        "wijk": "Wijk",
+        "kop-titel": "",
+        "helptekst": "Wij zijn bereikbaar van maandag tot en met vrijdag van 08.00 tot 18.00 uur.",
+        "helptekst-kop": "Lukt het niet om een melding te doen? Bel het telefoonnummer [070-373 8393](tel:+31703738393)",
+        "logo-omschrijving": "Logo gemeente",
```
Comment (alimpens) [inline src/shared/services/i18n/fallback/locales/nl/translations.json:8]: Do we want the keys to be kebab-case? Might be more difficult to work with in a js context, compared to camelCase for example

File: src/shared/services/i18n/fallback/locales/nl/translations.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        "ontevreden": "Nee, ik ben niet tevreden",
+        "ontevreden-waarom": "Waarover bent u niet tevreden?",
+        "ontevreden-meerkeuzes": "U kunt meer keuzes maken.",
+        "ontevreden-toelichting": "Wat wilt u ons hierover nog meer laten weten?",
+        "contact": "Contact",
+        "contact-waarom": "Uw reactie is belangrijk voor ons. Wij laten u graag weten wat wij er mee doen. En misschien willen wij u nog iets vragen of vertellen. Wij bellen u dan of sturen een e-mail.",
+        "contact-geen": "Nee, bel of e-mail mij niet meer over deze melding of over mijn reactie."
+   }
+
+  }
```
Comment (alimpens) [inline src/shared/services/i18n/fallback/locales/nl/translations.json:63]: Add newline after this line

File: src/shared/services/i18n/fallback/locales/nl/translations.json (hunk 1 above)
Comment (vdegraaf) [reply src/shared/services/i18n/fallback/locales/nl/translations.json:5]: I guess. It might be used by other municipalities

**Comment by vdegraaf** [comment]
@alimpens I changed the json file to follow conventions. In the next story we will need to create a mapper that maps the keys to understandable Dutch keys.

**Review by alimpens — APPROVED** [review]

**Pull request merged**